docs/DeveloperGuide.md

```diff
@@ -290,28 +290,32 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 (For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
 
-**Use case: Delete a person**
+
+**Use case: UC2 - List all people saved in `BizBook`**
 
 **MSS**
 
-1.  User requests to list persons
-2.  AddressBook shows a list of persons
-3.  User requests to delete a specific person in the list
-4.  AddressBook deletes the person
+1.  User requests to list all people saved in their `BizBook`
+2.  `BizBook` shows a list of persons
 
     Use case ends.
 
 **Extensions**
 
-* 2a. The list is empty.
+* 1a. If there are no contacts stored in the system.
+    * 1a1 `BizBook`shows an error message.
+
+      Use case ends.
+
 
-  Use case ends.
+* *a. `BizBook` detects an error in the entered command.
 
-* 3a. The given index is invalid.
+    * *a1 `BizBook`  requests for a valid command.
+    * *a2 User enters new command.
 
-    * 3a1. AddressBook shows an error message.
+      Steps *a1-*a2 are repeated until a valid command is input.
 
-      Use case resumes at step 2.
+      Use case ends.
```

Comment:
Should this be a resumes at step 2? Since the list of contact is not shown yet?

Reply:
I think I updated it before your comment? Because i deleted the "resumes at step 2"

Reply:
What i meant to say is should it remain as "resume at step 2"? Since this extension would return back to the MSS after a user inputs the right command.